pkg/controller/gittrack/handler_test.go

```diff
@@ -326,6 +577,55 @@ var _ = Describe("Handler Suite", func() {
 
 				AssertInvalidSubPath(kind)
 			})
+
+			PContext("when a child is owner by another controller", func() {
```

Comment:
The ignore logic is not yet implemented to handle the differences between `GitTrack` and `ClusterGitTrack` right now, so this must be pending